README.md

```diff
@@ -24,6 +24,14 @@ There is also a `build-arm.sh` script if you want to run the exporter on an ARMv
 
 This application tries to get data from the NetAtmo API. For that to work you will need to create an application in the [NetAtmo developer console](https://dev.netatmo.com/dev/myaccount), so that you can get a Client ID and secret.
 
+### Passing secrets
```

Comment:
I think this makes more sense as a subsection of "Usage" explaining how to get the credentials into the running process with either environment variables or CLI-arguments.